src/core/context/AppContextProvider.tsx

```diff
@@ -303,13 +335,6 @@ export const AppContextProvider = (props: {
     }
   }, [toolCallStreamStore]);
 
-  const handleReset = useCallback(() => {
```

Comment:
This was not being used anymore (used to be the old trash icon before chat history's handler). Removed it to avoid confusion around systemPrompt reset